doc/user/content/sql/create-materialized-view.md

```diff
@@ -1,78 +1,92 @@
+**OR REPLACE** | If a materialized view exists with the same name, replace it with the view defined in this statement. You cannot replace views that other views depend on, nor can you replace a non-view object with a view.
+**IF NOT EXISTS** | If specified, _do not_ generate an error if a materialized view of the same name already exists. <br/><br/>If _not_ specified, throw an error if a view of the same name already exists. _(Default)_
+_view&lowbar;name_ | A name for the materialized view.
 **(** _col_ident_... **)** | Rename the `SELECT` statement's columns to the list of identifiers, both of which must be the same length. Note that this is required for statements that return multiple columns with the same identifier.
-_cluster&lowbar;name_ | The cluster to maintain this materialized view. If not provided, uses the session’s cluster variable.
-_select&lowbar;stmt_ | The [`SELECT` statement](../select) whose output you want to materialize and maintain.
+_cluster&lowbar;name_ | The cluster to maintain this materialized view. If not specified, defaults to the active cluster.
+_select&lowbar;stmt_ | The [`SELECT` statement](../select) whose results you want to maintain incrementally updated.
 
 ## Details
 
-### Memory
+### Usage patterns
+
+Maintaining a materialized view in durable storage has resource and latency
+costs that should be carefully considered depending on the main usage of the
+view. It's a good idea to create a materialized view if:
 
-Views are maintained in memory. Because of this, one must be sure that all
-intermediate stages of the query, as well as its result set can fit in the
-memory of a single machine, while also understanding the rate at which the
-query's result set will grow.
+* The results need to be available across clusters;
+* View maintenance and query serving would benefit from being scaled
+  independently;
+* The final consumer of the view is a [subscription](../tail).
 
-For more detail about how different clauses impact memory usage, check out our
-[`SELECT`](../select) documentation.
+On the other hand, if you only need to access a view from a single cluster, you
+should consider creating a [non-materialized view](../create-view) and building
+an index on it instead. The index will incrementally maintain the results of
+the view updated in memory within that cluster, allowing you to avoid the costs
+and latency overhead of materialization.
+
+[//]: # "TODO(morsapaes) Point to relevant architecture patterns once these
+exist+add sinks to the mix once these are supported (if relevant)."
 
 ### Indexes
 
-A brief mention on indexes: Materialize automatically creates an in-memory index
-which stores all columns in the `SELECT` query's result set; this is the crucial
-structure that the view maintains to provide low-latency access to your query's
-results.
+Although you can query a materialized view directly, these queries will be
+issued against Materialize's storage layer. This is expected to be fast, but
```

Comment:
I'd also like to mention indexes in the context of chaining materialized views, but didn't really find a way to plug it in without having to write a whole lot more.